### PR TITLE
fix: ubuntu fixes

### LINKED ISSUE #1519: Failing of Tests & Builds on ubuntu
**Describe the bug**
Builds are failing on ubuntu VM for version 20.08.

**Expected behavior**
> A clear and concise description of what you expected to happen.

**Additional context**
> Specifically the workflows are failing constantly

### PR DESCRIPTION
**Pull request by Sloox**
Fixes #1519 

The workflow builds dont fail.

### CODE REVIEW

File: .github/workflows/ubuntu-workflow.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-18.04
```
Comment (piotradamczyk5) [inline .github/workflows/ubuntu-workflow.yml:13]: then we could have issues with performance tests

Comment (Sloox) [reply .github/workflows/ubuntu-workflow.yml:13]: @piotradamczyk5 will test again a few more things

Comment (piotradamczyk5) [reply .github/workflows/ubuntu-workflow.yml]: anyway this blocks us, so it is better to merge and once again fix the performance test issue

Comment (Sloox) [reply .github/workflows/ubuntu-workflow.yml]: tried with latest too

**Comment by Sloox** [comment]
@flank-it

File: .github/workflows/ubuntu-workflow.yml (hunk 1 above)
Comment (piotradamczyk5) [reply .github/workflows/ubuntu-workflow.yml]: latest for now is 
```
Ubuntu
  18.04.5
  LTS
```
but it will be changed to 20.04 soon

**Comment by Sloox** [comment]
@flank-it

File: .github/workflows/ubuntu-workflow.yml (hunk 1 above)
Comment (pawelpasterz) [reply .github/workflows/ubuntu-workflow.yml]: The latest build passes on the current configuration https://github.com/Flank/flank/pull/1521/checks?check_run_id=1737489139
Maybe that was just a temporary issue?

**Comment by Sloox** [comment]
@pawelpasterz @piotradamczyk5 @adamfilipow92 the build is begining to pass. It appears to have been a temporary issue.

**Pull request closed without merging**